docs/iris/src/whatsnew/latest.rst

```diff
@@ -24,8 +24,8 @@ Features
   associated with an axis over which the statistic is being run.
 
 * Supporting ``Iris`` for both ``Python2`` and ``Python3`` resulted in pinning our
-  dependency on `Matplotlib`_ at ``v2.x``.  Now that ``Python2`` support has
-  been dropped, ``Iris`` is free to use the latest version of `Matplotlib`_.
+  dependency on `matplotlib`_ at ``v2.x``.  Now that ``Python2`` support has
+  been dropped, ``Iris`` is free to use the latest version of `matplotlib`_.
```

Comment:
Capitalisation of Matplotlib is correct based on all their public facing text. I thought this had already been corrected. Rebasing problem?